src/server/scripts/EasternKingdoms/BlackrockMountain/BlackrockSpire/instance_blackrock_spire.cpp

```diff
@@ -283,6 +287,11 @@ class instance_blackrock_spire : public InstanceMapScript
                         if (GetBossState(DATA_DRAGONSPIRE_ROOM) != DONE)
                             Events.ScheduleEvent(EVENT_DARGONSPIRE_ROOM_STORE, 1000);
                     }
+                    break;
+                case DATA_SPAWN_FINKLE_EINHORN:
+                    if (TempSummon* finkle = instance->SummonCreature(NPC_FINKLE_EINHORN, finkleSpawnPosition))
```

Comment:
https://woehead.way-of-elendil.fr/?spell=16710

Reply:
(http://www.wowhead.com/spell=16710/finkle-is-einhorn)